### PR TITLE
[4.1] Testing sample data feeds

### PR DESCRIPTION
**Pull request by brianteeman**
Updates the feeds in the testing sampledata plugin as feeds.joomla.org is no more. Used the feeds list from https://www.joomla.org/rss.html

PR for #35461

Code review should be enough its a basic change

### CODE REVIEW

File: plugins/sampledata/testing/testing.php [hunk 1]
```diff
@@ -1429,22 +1429,22 @@ public function onAjaxSampledataApplyStep6()
...
 			array(
 				'name'     => Text::_('PLG_SAMPLEDATA_TESTING_SAMPLEDATA_NEWSFEEDS_NEWSFEED_1_NAME'),
-				'link'     => 'http://feeds.joomla.org/JoomlaExtensions',
+				'link'     => 'https://extensions.joomla.org/browse/new?format=feed&type=rss',
 				'ordering' => 4,
 			),
 			array(
 				'name'     => Text::_('PLG_SAMPLEDATA_TESTING_SAMPLEDATA_NEWSFEEDS_NEWSFEED_2_NAME'),
-				'link'     => 'http://feeds.joomla.org/JoomlaSecurityNews',
+				'link'     => 'https://developer.joomla.org/security-centre.feed?type=rsss',
```
Comment (Quy) [inline plugins/sampledata/testing/testing.php]:
```suggestion
				'link'     => 'https://developer.joomla.org/security-centre.feed?type=rss',
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 040a7468943a7f46b5dfe90a81bad08f407b7a5e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37136">issues.joomla.org/tracker/joomla-cms/37136</a>.</sub>

**Review by prakhar3062 — APPROVED** [review]

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 040a7468943a7f46b5dfe90a81bad08f407b7a5e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37136">issues.joomla.org/tracker/joomla-cms/37136</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37136">issues.joomla.org/tracker/joomla-cms/37136</a>.</sub>

**Comment by Quy** [comment]
Thanks.

**Comment by brianteeman** [comment]
thx

**Pull request merged**